ui/apps/platform/src/Containers/VulnMgmt/List/Images/VulnMgmtListImages.js

```diff
@@ -39,7 +39,7 @@ export const defaultImageSort = [
 ];
 
 export function getCurriedImageTableColumns(watchedImagesTrigger, isFeatureFlagEnabled) {
-    const isFrontendVMUpdatesEnabled = isFeatureFlagEnabled('ROX_FRONTEND_VM_UPDATES');
+    const isFrontendVMUpdatesEnabled = isFeatureFlagEnabled('ROX_POSTGRES_DATASTORE');
```

Comment:
Ooops, we were checking the flag twice in this file.

Reply:
discussed offline. Won't make changes because that cannot be refactored outside of callback below